code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -203,7 +205,7 @@
 			mob_occupant.Unconscious((mob_occupant.bodytemperature * unconscious_factor) * 2000)
 		if(beaker)
 			if(reagent_transfer == 0) // Magically transfer reagents. Because cryo magic.
-				beaker.reagents.trans_to(occupant, 1, efficiency * 0.25) // Transfer reagents.
+				beaker.reagents.trans_to(occupant, 1, efficiency * 25) // Transfer reagents.
```

Comment:
See: tgstation/tgstation#32565
You should set this to a more modest number, like 1 or 2. (btw that still won't keep up with medicines that gets metabolized quickly; iirc 1~2 is the region where cryoxadone might or might not run out)

Reply:
Yeah it seems a bit much. I'm not sure if it's an oversight on op behalf. You're getting like 73u from 1u of cryoxadone on lowest efficiency. Not to mention to OD potential.

Reply:
@Rukofamicom in your med rework, do you plan to change reagent processing to stop players from loading up on chems?

Reply:
@park66665 actually going to use this instead [](https://github.com/tgstation/tgstation/pull/56755). Removes the reagents_transfer counter but I may need to tweak the maths as our cryo isn't the same as tg's